---
abstract: Population genomic data has revealed patterns of genetic variation associated
  with adaptation in many taxa. Yet understanding the adaptive process that drives
  such patterns is challenging; it requires disentangling the ecological agents of
  selection, determining the relevant timescales over which evolution occurs, and
  elucidating the genetic architecture of adaptation. Doing so for the adaptation
  of hosts to their microbiome is of particular interest with growing recognition
  of the importance and complexity of hostâ€“microbe interactions. Here, we track the
  pace and genomic architecture of adaptation to an experimental microbiome manipulation
  in replicate populations of Drosophila melanogaster in field mesocosms. Shifts in
  microbiome composition altered population dynamics and led to divergence between
  treatments in allele frequencies, with regions showing strong divergence found on
  all chromosomes. Moreover, at divergent loci previously associated with adaptation
  across natural populations, we found that the more common allele in fly populations
  experimentally enriched for a certain microbial group was also more common in natural
  populations with high relative abundance of that microbial group. These results
  suggest that microbiomes may be an agent of selection that shapes the pattern and
  process of adaptation and, more broadly, that variation in a single ecological factor
  within a complex environment can drive rapid, polygenic adaptation over short timescales.
authors:
- Seth M. Rudman
- Sharon Greenblum
- Rachel C. Hughes
- Subhash Rajpurohit
- Ozan Kiratli
- Dallin B. Lowder
- Skyler G. Lemmon
- Dmitri A. Petrov
- John M. Chaston
- Paul Schmidt
date: "2019-09-16"
header:
  caption: ""
  image: IMG_0821_small.jpg
highlight: false
image_preview: ""
math: true
projects:
- host-microbiome
- genomics-ecology
publication: Proceedings of the National Academy of Sciences
publication_short: ""
publication_types:
- "2"
selected: true
title: Microbiome composition shapes rapid genomic adaptation of Drosophila melanogaster
url_code: ""
url_custom:
- name: DOI
  url: https://doi.org/10.1073/pnas.1907787116
url_dataset: https://www.pnas.org/highwire/filestream/889054/field_highwire_adjunct_files/1/pnas.1907787116.sd01.xlsx
url_pdf: https://www.pnas.org/content/pnas/early/2019/09/12/1907787116.full.pdf
url_poster: ""
url_preprint: ""
url_project: ""
url_slides: ""
url_source: ""
url_video: ""
---

Microbiomes have profound effects on many aspects of the biology of their hosts. The powerful and widespread nature of these effects has lead to speculation that microbiomes may shape the evolutionary trajectories of host populations. In this study we used a large outdoor experiment to manipulate the microbiome composition of replicate populations of *Drosophila* to directly test whether divergence in microbiome composition drives divergent rapid adaptation. Using population genomic data we demonstrate that microbiomes drive divergence at dozens of loci genome wide. Moverover, the direction of allele frequency change at these loci mirrors that observed in natural populations that vary in their microbiome composition. Together, these results convincingly demonstrate the powerful effect microbiomes can have on host evolution.


## Significance

Natural selection can drive evolution over short timescales. However, there is little understanding of which ecological factors are capable of driving rapid evolution and how rapid evolution alters allele frequencies across the genome. Here, we combine a field experiment with population genomic data from natural populations to assess whether and how microbiome composition drives rapid genomic evolution of host populations. We find that differences in microbiome composition cause divergence in allele frequencies genome-wide, including in genes previously associated with local adaptation. Moreover, we observed concordance between experimental and natural populations in terms of the direction of allele frequency change, suggesting that microbiome composition may be an agent of selection that drives adaptation in the wild.
